### ISSUE TITLE
Support annotations for the Service resource

### ISSUE THREAD

**Issue by ilyas1uphealth** [trinodb/charts#145]
Edit: Created a PR: https://github.com/trinodb/charts/pull/149

When using aws-load-balancer-controller, it is useful to pass annotations in the Service ( https://github.com/trinodb/charts/blob/trino-0.19.0/charts/trino/templates/service.yaml ) to denote what properties the load balancer could have. 

Proposed solution:
From https://github.com/trinodb/charts/blob/trino-0.19.0/charts/trino/values.yaml#L226
```
service:
  type: ClusterIP
  port: 8080
  annotations: {} # <--------- this field is new
  ```
  From  https://github.com/trinodb/charts/blob/trino-0.19.0/charts/trino/templates/service.yaml 
  ```
  metadata:
  name: {{ template "trino.fullname" . }}
  labels:
    app: {{ template "trino.name" . }}
    chart: {{ template "trino.chart" . }}
    release: {{ .Release.Name }}
    heritage: {{ .Release.Service }}
  ####### 
  # This section is new
  ########
  {{- with .Values.service.annotations }}
  annotations:
    {{- toYaml . | nindent 4 }}
  {{- end }}
  ####### 
  # /This section is new
  ########
spec:
```

I could make a pull request if you'd like.

**Comment by ilyas1uphealth** [comment]
Created a PR: https://github.com/trinodb/charts/pull/149

**Comment by nineinchnick** [comment]
Fixed by #134
